cypress/support/api_commands.js

```diff
@@ -0,0 +1,10 @@
+Cypress.Commands.add('api_getCustomers', ({ page = 1, limit = 10, size = 'All', failOnStatusCode = true } = {}) => {
+  const queryParams = { page, limit, size }
+  const requestOptions = {
+    method: 'GET',
+    url: `${Cypress.env('API_URL')}/customers`,
+    qs: queryParams,
+    failOnStatusCode: failOnStatusCode
```

Comment:
Quando do parâmetro/variável tem o mesmo nome do propriedade, este pode ser omitido.

```suggestion
    failOnStatusCode
```